src/components/Modals/Send/hooks/useFormSend/useFormSend.test.tsx

```diff
@@ -53,33 +53,33 @@ const formData: SendInput = {
     explorerAddressLink: 'https://etherscan.io/address/',
   },
   [SendFormFields.AmountFieldError]: '',
-  [SendFormFields.FeeType]: chainAdapters.FeeDataKey.Average,
+  [SendFormFields.FeeType]: FeeDataKey.Average,
   [SendFormFields.EstimatedFees]: {
-    [chainAdapters.FeeDataKey.Slow]: {
+    [FeeDataKey.Slow]: {
       txFee: '3100000000000000',
       chainSpecific: {
         gasLimit: '42000',
         gasPrice: '10000000000',
         satoshiPerByte: '5',
       },
     },
-    [chainAdapters.FeeDataKey.Average]: {
+    [FeeDataKey.Average]: {
       txFee: '3100000000000000',
       chainSpecific: {
         gasLimit: '42000',
         gasPrice: '10000000000',
         satoshiPerByte: '5',
       },
     },
-    [chainAdapters.FeeDataKey.Fast]: {
+    [FeeDataKey.Fast]: {
       txFee: '3100000000000000',
       chainSpecific: {
         gasLimit: '42000',
         gasPrice: '10000000000',
         satoshiPerByte: '5',
       },
     },
-  },
+  } as { [k in FeeDataKey]: FeeData<ChainId> },
```

Comment:
I know this isn't directly related to your changes, but figure it would be good to shed some light on as we review these changes/updates

Reply:
Improved typings in 01c05c684a4f3943fe0951c0ae4e8d943b775b89 - agreed since this is adding a lot of casting, it's a step in the right direction that will make the types refactoring easier